src/ape/api/config.py

```diff
@@ -30,3 +30,8 @@ def serialize(self) -> Dict:
 
     def validate_config(self):
         pass
+
+
+class ConfigDict(ConfigItem):
+    def __post__init__(self):
```

Comment:
Should be `__post_init__` (one underscore in the middle)?

Reply:
```suggestion
    def __post_init__(self):
```